vmdb/app/models/vm_or_template.rb

```diff
@@ -1165,7 +1165,7 @@ def self.refresh_ems(vm_ids)
   end
 
   def refresh_ems
-    raise "no #{ui_lookup(:table => "ext_management_systems")} or credentials defined" unless self.ext_management_system && self.ext_management_system.authentication_valid?
+    raise "no #{ui_lookup(:table => "ext_management_systems")} or credentials defined" unless self.ext_management_system && self.ext_management_system.has_credentials?
```

Comment:
Another place to use authentication_status_ok?  I don't want to refresh if the system not reachable or our password is bad.